client/app/src/accounts/account.controller.js

```diff
@@ -98,6 +99,17 @@
 
     pluginLoader.triggerEvent('onStart')
 
+    electron.ipcRenderer.on('uri', (event, uri) => {
+      $timeout(() => {
+        const qrcodeElement = document.querySelector('ark-qrcode')
+        const scheme = qrcodeElement.deserializeURI(uri)
+
+        if (!scheme) return toastService.error('Invalid URI')
+        if (scheme && !self.selected) return toastService.error('Select an account and open the URI again')
```

Comment:
How about a dialog to select an account?

Reply:
would be great! however it will be pending for next versions, I will open an issue to handle this and the unit tests